### PR TITLE
Added Wishlist

### PR DESCRIPTION
**Pull request by TusharJade**
### Added Wishlist pag
Screenshot (36)
e

### CODE REVIEW

**Comment by sashaboi** [comment]
I dont know what element is causing this , but the cursor seems to turn to pointer not just on the card but a little bit outside it as well, could that be the issue of some margins/paddings conflict? 
hope you get a chance to look at it , 

Also the cross button:hover cursor might need to be pointer as well.


image

Onkar Deshpande

**Comment by TusharJade** [comment]
> I dont know what element is causing this , but the cursor seems to turn to pointer not just on the card but a little bit outside it as well, could that be the issue of some margins/paddings conflict? hope you get a chance to look at it ,
> 
> Also the cross button:hover cursor might need to be pointer as well.
> 
> image
> 
> Onkar Deshpande

Got it, thanks for pointing it out Onkar will fix all that now

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
             <a class="landing-a" href="#">Cart Management</a>
         </li>
         <li class="landing-li">
-            <a class="landing-a" href="#">Wishlist Management</a>
+            <a class="landing-a" href="/HTML and CSS/Wishlist/wishlist.html">Wishlist Management</a>
```
Comment (harshitpaliwal95) [inline index.html]: - good classes names
- looks good

**Comment by TusharJade** [comment]
> I dont know what element is causing this , but the cursor seems to turn to pointer not just on the card but a little bit outside it as well, could that be the issue of some margins/paddings conflict? hope you get a chance to look at it ,
> 
> Also the cross button:hover cursor might need to be pointer as well.
> 
> image
> 
> Onkar Deshpande

Hey @sashaboi , first of all thank you so much for pointing it out that was happening because I copy pasted cards from product listing page but now I fixed that please have a look

File: index.html (hunk 1 above)
Comment (TusharJade) [reply index.html]: index.html is just have links of pages like wishlist, home page and will be deleting that page after some time thats why I dont focus on class name but will fix that now thanks for pointing it out

Comment (TusharJade) [reply index.html]: Hey @harshitpaliwal95 gave good class name please have a look

**Pull request merged**